src/mumble/Settings.h

```diff
@@ -290,6 +290,15 @@ struct Settings {
 	QString qsProxyHost, qsProxyUsername, qsProxyPassword;
 	unsigned short usProxyPort;
 
+	/// iMaxInFlightTCPPings specifies the maximum
+	/// number of ping messages that the client has
+	/// sent, but not yet recieved a response for
+	/// from the server. This value is checked when
+	/// the client sends its next ping message. If
```

Comment:
Maybe we should make it clear that this is about periodic pings? Also in the commit message.

Although we probably don’t do so right now, pings could also be sent in other cases, where this count should not apply.